eng/nuget/Microsoft.NET.Workload.Emscripten.Manifest/WorkloadManifest.json.in

```diff
@@ -2,7 +2,6 @@
   "version": "${WorkloadVersion}",
   "workloads": {
     "microsoft-net-sdk-emscripten": {
-      "abstract": true,
```

Comment:
is this needed?

Reply:
Is there a problem w/ abstract workloads and installers?

Reply:
That doesn't sound right. I built a copy of EMSDK locally. Abstract workloads are turned into componentgroups in VS, but as long as they have packs, it should produce it, e.g.

```
"id": "microsoft.net.sdk.emscripten",
      "version": "6.0.0.0",
      "type": "Component",
      "isUiGroup": true,
      "dependencies": {
        "Microsoft.Emscripten.Node.6.0.0-preview.7.21358.1": {
          "version": "6.0.0.0",
          "behaviors": "IgnoreApplicabilityFailures"
        },
        "Microsoft.Emscripten.Python.6.0.0-preview.7.21358.1": {
          "version": "6.0.0.0",
          "behaviors": "IgnoreApplicabilityFailures"
        },
        "Microsoft.Emscripten.Sdk.6.0.0-preview.7.21358.1": {
          "version": "6.0.0.0",
          "behaviors": "IgnoreApplicabilityFailures"
        }
```

Reply:
```
  Creating nuget packages...
  Microsoft.NET.Runtime.Emscripten.Node -> C:\source\repos\dotnet\emsdk\artifacts\packages\Debug\Shipping\Microsoft.NET.Runtime.Emscripten.2.0.23.Node.win-x64.6.0.0-dev.nupkg
  Microsoft.NET.Runtime.Emscripten.Python -> C:\source\repos\dotnet\emsdk\artifacts\packages\Debug\Shipping\Microsoft.NET.Runtime.Emscripten.2.0.23.Python.win-x64.6.0.0-dev.nupkg
  Microsoft.NET.Runtime.Emscripten.Sdk -> C:\source\repos\dotnet\emsdk\artifacts\packages\Debug\Shipping\Microsoft.NET.Runtime.Emscripten.2.0.23.Sdk.win-x64.6.0.0-dev.nupkg
  Microsoft.NET.Workload.Emscripten.Manifest -> C:\source\repos\dotnet\emsdk\artifacts\packages\Debug\Shipping\Microsoft.NET.Workload.Emscripten.Manifest-6.0.100.6.0.0-dev.nupkg
  875
  microsoft-net-sdk-emscripten is abstract and will be skipped.
C:\source\repos\dotnet\emsdk\eng\workloads\workloads.csproj(115,5): error MSB4044: The "CreateLightCommandPackageDrop" task was not given a value for the required parameter "InstallerFile".

Build FAILED.

C:\source\repos\dotnet\emsdk\eng\workloads\workloads.csproj(115,5): error MSB4044: The "CreateLightCommandPackageDrop" task was not given a value for the required parameter "InstallerFile".
    0 Warning(s)
    1 Error(s)
```

Note the `  microsoft-net-sdk-emscripten is abstract and will be skipped.`

Reply:
It looks like you're on a very old build of the Arcade tooling then. Abstract workloads with packs will produce installers.